### PR TITLE
cf-1826D implem

### PR DESCRIPTION
**Pull request by envyaims**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by envyaims** [comment]
No Java because i hate java
https://codeforces.com/contest/1826/submission/204672085
https://codeforces.com/contest/1826/submission/227337626

File: solutions/silver/cf-1826D [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main(){
+	int T;
+	cin >> T;
+	while(T--){
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1826D:24]:
```suggestion
	int test_num;
	cin >> test_num;
	for (int t = 0; t < test_num; t++) {
```

File: solutions/silver/cf-1826D [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		int N;
+		cin >> N;
+		vector<int> A(N);
+		for(int i = 0; i < N; i++) cin >> A[i];
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1826D:28]: lowercase

File: solutions/silver/cf-1826D [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	}
+}
+```
+
+</CPPSection>
+
+<PySection>
+
+```py
+from sys import stdin, stdout
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1826D:57]: is `input()` too slow

Comment (SansPapyrus683) [reply solutions/silver/cf-1826D:57]: also tabs 💀

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/silver/cf-1826D (hunk 3 above)
Comment (envyaims) [reply solutions/silver/cf-1826D:57]: THE BOTS LACKING

Comment (envyaims) [reply solutions/silver/cf-1826D:57]: > is `input()` too slow

develop good habits

**Review by ryanchou-dev — APPROVED** [review]
i think `input()` might be better for python js because it should be sufficient for bronze/silver

but if you'd add a comment to explain it, that would also work :D

File: solutions/silver/cf-1826D (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/silver/cf-1826D]: convention

Comment (SansPapyrus683) [reply solutions/silver/cf-1826D]: convention

**Pull request merged**